src/components/MySpace/Tools/NoDeveloperAccess.tsx

```diff
@@ -3,20 +3,45 @@ import { Link } from "react-router-dom";
 import { useTranslation } from "react-i18next";
 import NoDevAccessImg from "images/no_developer_access.svg";
 
-function NoDeveloperAccess() {
+function NoDeveloperAccess({ access }: { access?: string }) {
   const { t } = useTranslation();
 
-  return (
-    <>
-      <div className="w-100 py-5 text-center">
-        <img className="d-inline" src={NoDevAccessImg} alt="no wizard" />
-        <h5>{t("mySpace.myTools.noAccessHeader")}</h5>
+  const renderBody = () => {
+    if (access === "pending") {
+      return (
+        <p>Developer access request is pending</p>
+      )
+    }
+    if (access === "rejected") {
+      return (
+        <>
+          <p>Developer access request has been rejected</p>
+          <Button>
+            <Link to="/my-space/request/developer">
+              {t("profile.requestDevAccess")}
+            </Link>
+          </Button>
+        </>
+      )
+    }
+    else return (
```

Comment:
is `else return` required can't we do `return` only

Reply:
True, on it